lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -39,14 +39,24 @@ class GoogleCloudOutput < BufferedOutput
     Fluent::Plugin.register_output('google_cloud', self)
 
     PLUGIN_NAME = 'Fluentd Google Cloud Logging plugin'
-    PLUGIN_VERSION = '0.5.4.grpc.alpha.1'
+    PLUGIN_VERSION = '0.5.5.v2.alpha.1'
 
     # Constants for service names.
     APPENGINE_SERVICE = 'appengine.googleapis.com'
```

Comment:
I'm wondering if it's better to have a single constant for each service/resource, rather than two parallel sets of variables, e.g.:
```
APPENGINE_CONSTANTS = {service: 'appengine.googleapis.com', resource_type: 'gae_app'}
CLOUDFUNCTIONS_CONSTANTS = {service: 'cloudfunctions.googleapis.com', resource_type: 'cloud_function'}
COMPUTE_CONSTANTS = {service: 'compute.googleapis.com', resource_type: 'gce_instance'}
...
```
and then reference them as `APPENGINE_CONSTANTS[:service]` and `APPENGINE_CONSTANTS[:resource_type]`, respectively?